test/integration/getEntries.test.ts

```diff
@@ -334,15 +334,16 @@ describe('getEntries via client chain modifiers', () => {
         include: 5,
         'sys.id': entryWithResolvableLink,
       })
-      assertLocalizedEntriesResponse(response)
+      assertLocalizedEntriesResponse(response, true)
```

Comment:
this is now always called with `true`, should we remove the second arg?

Reply:
Nope, on `enforces entry.sys when query.select is defined` test is false.

Reply:
isn't it a different function, `assertCSMEntriesResponse`?